src/Core/View.js

```diff
@@ -282,6 +282,27 @@ View.prototype.addLayer = function addLayer(layer, parentLayer) {
 };
 
 /**
+ * Removes a layer from the view
+ * @param {string} layerId - the id of the layer to remove
+ */
+View.prototype.removeLayer = function removeLayer(layerId) {
+    const layers = this.getLayers((l => l.id == layerId));
+    if (layers.length === 0) {
+        throw new Error(`No layer with id '${layerId}' to remove`);
+    }
+    const toRemove = this.scene.children.filter((c => c.layer == layerId));
```

Comment:
If I'm not mistaken, obj are not necessarily added to `this.scene` any more (see GeometryLayer constructor).

Reply:
True. I should use `layer.object3d` instead of `this.scene`.